pkg/providers/tinkerbell/template.go

```diff
@@ -362,6 +362,11 @@ func buildTemplateMapCP(
 	etcdTemplateOverride string,
 	datacenterSpec v1alpha1.TinkerbellDatacenterConfigSpec,
 ) (map[string]interface{}, error) {
+	auditPolicy, err := common.GetAuditPolicy(clusterSpec.Cluster.Spec.KubernetesVersion)
```

Comment:
then you can move `values["auditPolicy"] = auditPolicy` to https://github.com/aws/eks-anywhere/blob/b1d360e9b52c030136d2d961b1f1756c29165852/pkg/providers/tinkerbell/template.go#L386

Reply:
yes